### PR TITLE
Create app skeleton

### PR DESCRIPTION
**Pull request by KarthikRaju391**
###  Following changes have been implemented as per the acceptance criteria

- installed fonts 
- reset default browser styles with `styled-normalize`
- add global styles
- add router with routes `/` and `/search`

### CODE REVIEW

**Comment by mentor-tara** [comment]
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime **I'd suggest you start with the next task already. Even if you just check out a new branch** where you can continue to work on later. This is a great opportunity to learn professional Git workflows. 

[Head back to the course material to learn how to proceed.](https://profy.dev/project/reddit-timer/while-waiting)

PS: Of course we all know I'm not busy. I'm a bot :robot: But let's pretend that I am to have a more realistic team environment. We're here to practice after all, right? :smile:[](ckkp04mmw2vuk0b42idpakyah)

**Review by mentor-tara — CHANGES_REQUESTED** [review]
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [x] **Use yarn instead of npm** [:link:](https://profy.dev/project/reddit-timer/review-tips/use-yarn)
- [x] **Use a common project structure** [:link:](https://profy.dev/project/reddit-timer/review-tips/file-structure)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/create-app-skeleton-d137af0d-ce79-4289-ae9b-df6f0444a380). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckkp0fo0g2w0w0b422db33fjd)

**Review by mentor-tara — APPROVED** [review]
Great work! :tada::tada::tada: You can merge the PR now.[](ckkqnj5io3o540b425ilxp80y)

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 
 function App() {
   return (
-    <div>
-      App Placeholder
-    </div>
+    <Router>
+      {/* <div>
+        App Placeholder
+      </div> */}
```
Comment (jkettmann) [inline src/App.js:11]: The code looks very clean. I'd only remove this comment

**Comment by KarthikRaju391** [comment]
Just out of curiosity, is it really you responding or is it a bot 😬

**Comment by jkettmann** [comment]
That was really me. My bots aren't that smart yet 🤓

**Comment by KarthikRaju391** [comment]
Haha, thanks!

**Pull request merged**